### DISCUSSION TITLE
Second Cover page -Add Examination committee

### DISCUSSION THREAD

**Discussion by GIGASHAB** [joaomlourenco/novathesis#432] [Q&A] [answered]
Hi, I would like to add the examination committee name in second cover page ; I already edit the names and removed "%" to code be run. but still any of the names is appears. How can I solve it?

**Comment by joaomlourenco** [comment, marked as answer]
Is this problem with the Word or the LaTeX template?  You're posting this in the Word project.

But, if it is with the LaTeX template, you have to change the document status for working/provisional to final.

**Comment by GIGASHAB** [comment]
Thanks for the prompt reply, in LaTex template 

I changed in Config/1_novathesis.tex. Still is not appearing

**Comment by GIGASHAB** [comment]
Thanks, Solve :)
